### DISCUSSION TITLE
Docker compose Qnap

### DISCUSSION THREAD

**Discussion by RobertVilaplana** [Chia-Network/chia-blockchain#2618] [General]
Hello

I am going to try to start a node for Chia in a Qnap server that I already have running and where I am going to expand hdd disks.

Is it possible to use docker-compose to start the application?
After installation through docker, is there an interface for configuration ?, or do you have to configure through yaml files?

Finally, for the generation of the temporary files, I understand that I must install a ssd on the server for the creation of plots, but its capacity for several K32 plots must be at least 250 Gb or more. I understand that the temporary creation file is deleted after each plot created?

Thanks

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
